### PR TITLE
April update first draft

### PR DESCRIPTION
**Pull request by mrmrcoleman**

### CODE REVIEW

File: blog/_posts/2019-04-05-april-monthly-update.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+#### Florian Pichler joins simplabs
+
+We're very excited to announce that [Florian Pichler](https://twitter.com/pichfl/) has joined simplabs.
+
+<INSERT FLORIAN'S BACKGROUND INFORMATION HERE>
+
+Our CEO Marco says _"I'm very happy to have Florian on board. His expertise in both the engineering and design fields as well as experience working with big corporate clients make him a valuable addition to the team and will allow us to bring more value to more clients in the future."_
+
+#### simlabs welcomes new client Qonto
```
Comment (Turbo87) [inline blog/_posts/2019-04-05-april-monthly-update.md]:
```suggestion
#### simplabs welcomes new client Qonto
```

**Review by Turbo87 — COMMENTED** [review]
😉

File: blog/_posts/2019-04-05-april-monthly-update.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+## News
+
+#### Florian Pichler joins simplabs
+
+We're very excited to announce that [Florian Pichler](https://twitter.com/pichfl/) has joined simplabs.
+
+Florian’s career has varied from user interfaces and print design to frontend development but what ties his entire career together is a strong focus on user experience.
+
+He has a wealth of experience doing consulting work around new technologies and both native and web-based apps. In this work he has helped to shape, design and bring to life projects for Audi, BMW, Microsoft, Zurich and many more.
```
Comment (marcoow) [inline blog/_posts/2019-04-05-april-monthly-update.md]:
```suggestion
He has a wealth of experience doing consulting work around new technologies and both native and web-based apps. In this work he has helped to shape, design and bring to life projects for Audi, BMW, Microsoft, Zurich Insurance Group and many more.
```

not sure?

File: blog/_posts/2019-04-05-april-monthly-update.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+We're very pleased to be sponsoring [ElixirConfEU](http://www.elixirconf.eu/) which is starting in Prague on the 8th of April. We've sent over a load of simplabs stickers, so grab one while you can.
+
+![ElixirConfEU 2019](/images/posts/2019-04-05-april-monthly-update/elixir-conf-stickers.jpg)
+
+#### EmberFestEU 2019 tickets go on sale soon
+
+![EmberFestEU](/images/posts/2019-04-05-april-monthly-update/ember-fest-logo.png)
+
+Also, we've been busy behind the scenes getting ready for this year's [EmberFestEU](https://emberfest.eu/). Early bird tickets are going on sale soon so keep an eye on the [twitter](https://twitter.com/EmberFest) for more news.
```
Comment (marcoow) [inline blog/_posts/2019-04-05-april-monthly-update.md]:
```suggestion
Also, we've been busy behind the scenes getting ready for this year's [EmberFestEU](https://emberfest.eu/). We have booked the venue and will announce the host city soon - there are some hints on the website already though so go see whether you can find them. Early bird tickets are going on sale soon so keep an eye on the [twitter](https://twitter.com/EmberFest) for more news as well.
```

**Review by marcoow — COMMENTED** [review]
so much stuff again ❤️

File: blog/_posts/2019-04-05-april-monthly-update.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+
+The next DublinJS meetup, taking place in May and hosted by Chris Manson, is already [online](https://www.meetup.com/DublinJS/events/fbllfpyzhbkb/). Speakers to follow soon.
+
+The next Ember.js Dublin meetup will take place on April 18th will be an Ember Contributor workshop. Chris Manson will be running a "workshop to get you started contributing to one of the many Ember projects". Details [here](https://www.meetup.com/emberjsdublin/events/260148921/).
+
+## Past Events
+
+#### EmberConf 2019
+
+simplabs sent a group of people to EmberConf2019 in Portland where Ricardo Mendes hosted a contributor workshop and [Jessica Jordan](https://twitter.com/jjordan_dev) gave a talk about Crafting Web Comics in Ember.
```
Comment (marcoow) [inline blog/_posts/2019-04-05-april-monthly-update.md:69]: The video of jessica's talk has been released: https://www.youtube.com/watch?v=C_abEQENdy0

Comment (marcoow) [reply blog/_posts/2019-04-05-april-monthly-update.md:69]: Chris gave a mini talk as well btw that we should include here as well: https://www.youtube.com/watch?v=GHbjdesHhPA

**Pull request merged**